src/main/scala/uk/gov/hmrc/testreport/model/Violation.scala

```diff
@@ -22,7 +22,9 @@ case class Violation(
   impact: String,
   occurrences: List[Occurrence],
   exclusionRules: Set[ExclusionRule] = Set.empty
-)
+) {
+  val isExcluded: Boolean = exclusionRules.nonEmpty
```

Comment:
hmm, what about "has some that are excluded"?

for example with skip link we have "all page elements must be contained in landmark" accessibility check, which can get triggered on the skip-link, but also have other legitimate html trigger it, that people might need to address

Reply:
nevermind me, think I got it now 😵 

we process the axeviolations into included/excluded and then we group them after that into lists of violations

so isExcluded is right - because you couldn't have a Violation with a mix of excluded and included snippets in occurrences

is that right?

Reply:
yup, have followed the thread and refreshed myself - that is it - sorry for the misdirection

future further refactoring what-if'ing, it does feel like there's a lot happening in TestReportPlugin

though the more I read it repeatedly, it's not taken that many reads to get it into my brain 🤷

Reply:
Yeah the plugin class is quite [hadouken](https://www.reddit.com/r/ProgrammerHumor/comments/3hbov9/hadouken/)-y, but I didn't want to try refactor the world right now 😅